### DISCUSSION TITLE
Search for something

### DISCUSSION THREAD

**Discussion by tomyems** [UKHomeOffice/design-system#484] [General]
To discuss improvements and suggestions for the [search for something](https://design.homeoffice.gov.uk/patterns/search-for-something) pattern.

**Comment by atin33** [comment]
Noticed this feature, looks interesting. How can we use this in our own service. I didn't find the component/code anywhere.

**Reply by benjystanton** [reply]
Hi @atin33 did you manage to get any help with this question? It may be worth getting in touch with the HO design system working group on Slack. They may be able to point you towards some code examples if they exist.

**Reply by atin33** [reply]
Thanks @benjystanton. I haven't heard anything yet, will reach out to HO DS team.

**Comment by Phoebe189** [comment]
It would also be useful to standardise/set best practice across the Home Office for how we clear the search box. 
For example
- whether using the x in the input field to clear it is accessible or if we need a link underneath the search box instead/as well
- when we would clear existing search results as well as the search box text
- when we offer the option to replay previous searches to the user
